src/app/vault/vault-filter/vault-filter.module.ts

```diff
@@ -0,0 +1,48 @@
+import { NgModule } from "@angular/core";
```

Comment:
I would like to move this module into a new folder at `src/app/modules/vault-filter`. This is similar to what I've got right now on the desktop [PR](src/app/modules/vault-filter/vault-filter.module.ts). This is also something I can do next week. Any thoughts?

Reply:
Yep, that's fine! It made sense to me here in the vault context, but I think it works well there too

Reply:
Cool. I don't disagree that it works where it is, I just think moving away from having business verbiage as top level structures is in our best interests long term.

Reply:
Done